lib/pundit/authorization.rb

```diff
@@ -167,6 +167,18 @@ def pundit_user
       current_user
     end
 
+    # Hook method which allows to clear the Pundit context.
+    #
+    # @see https://github.com/varvet/pundit#additional-context
```

Comment:
I believe this should explain the purpose of this method, the concept of "Pundit context" isn't well-established enough to refer to.

More specifically it might be helpful to know _why_ one would call this. It might be prudent to specifically call out "when you change the `current_user`".

Additionally, the link here has no relevance at all to the method. That should be removed.